### PR TITLE
Ana mucibabic

### PR DESCRIPTION
**Pull request by ancimpek**
Hi Kristijan, I have finished the exercise, please review to complete the last task.

### CODE REVIEW

**Review by kristijan-ristic — COMMENTED** [review]
Once you resolve conflicts I will approve your PR

File: Candidates.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra your name goes here - replace second line with your name
-Ana Mucibabic Ana-Mucibabic
\ No newline at end of file
+Ana Mucibabic Ana-Mucibabic
```
Comment (kristijan-ristic) [inline Candidates.txt:4]: It's still not correct. You should (pull) have the latest master code on your local master branch in order to merge the latest master code with your branch

Comment (ancimpek) [reply Candidates.txt:4]: Is it now correct? I pulled the latest master code on my local master branch.

Comment (kristijan989) [reply Candidates.txt]: Now you have all changes, but you didn't merge it how it is expected. You should keep all three rows in that file and add your name on next free line

File: Candidates.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra Milenkovic Aleksandra-Milenkovic1
-Maja Dramicanin maja-dramicanin
+Ana Mucibabic Ana-Mucibabic
```
Comment (kristijan-ristic) [inline Candidates.txt]: That's fine, but you didn't merge this file as expected. You shouldn't remove some rows from that file, you should just add your at the end of this file (at next empty line)

Comment (ancimpek) [reply Candidates.txt]: Can you please review it again? I did another commit

**Review by kristijan-ristic — APPROVED** [review]

**Pull request merged**